### PR TITLE
Partial removal of pasteup

### PR DESCRIPTION
**Pull request by nicl**
## What does this change?

A first stab at removing `pasteup`. It's too big to remove in one go unfortunately.

This switches to src-foundations for typography. And also moves out icons and logos.

See https://github.com/guardian/dotcom-rendering/pull/842/commits/2c901c365b93b3c0fc1089501234e0b797237b90 for a mapping of old pasteup values to the new (closest) equivalent src-foundations ones. cc @oliverlloyd and @zeftilldeath .

Still to do:

* remaining palette stuff
* remaining media-query stuff

## Why?

`src-foundations` is the preferred source of truth here.

## Link to supporting Trello card

https://trello.com/c/psXEfcWL

cc also https://github.com/guardian/dotcom-rendering/pull/840 which @shtukas has done!

@zeftilldeath and @SiAdcock there isn't exact equivalence between pasteup font sizes and src-foundations. I'll sit down with you both to go over things, and see if the choices I've made (where they don't equate) are sensible or not. At the least, I suspect some of the line heights are not ideal.

To give a sense of the differences, here is AMP now:
amp-pasteup

and with these changes:
amp-src-foundations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,10 +128,10 @@
             "<rootDir>/scripts/jest/setup.ts"
         ],
         "moduleNameMapper": {
-            "^@root(.*)$": "<rootDir>$1",
-            "^@frontend(.*)$": "<rootDir>/packages/frontend$1",
             "^svgs/(.*)$": "<rootDir>/__mocks__/svgMock.tsx",
-            "^(.*)\\.svg$": "<rootDir>/__mocks__/svgMock.tsx"
+            "^(.*)\\.svg$": "<rootDir>/__mocks__/svgMock.tsx",
```
Comment (nicl) [inline package.json]: This is an ordering issue - now that the svgs live in the frontend package, we need to match them first to mock them in the tests.

**Comment by MatthewJWalls** [comment]
image

image

File: packages/frontend/amp/components/KeyEvents.tsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import { css } from 'emotion';
-import { headline, textSans } from '@guardian/pasteup/typography';
-import { palette } from '@guardian/src-foundations';
+import { palette, headline, textSans } from '@guardian/src-foundations';
 // import { pillarPalette } from '@frontend/lib/pillars';
-import DownArrow from '@guardian/pasteup/icons/down-arrow.svg';
+import DownArrow from '@frontend/static/icons/down-arrow.svg';
```
Comment (SiAdcock) [inline packages/frontend/amp/components/KeyEvents.tsx]: I think some of these SVGs are in the [`src-svgs` package](https://github.com/guardian/source-components/tree/master/packages/svgs) if that's helpful?

https://www.figma.com/file/Ai7AELHC6KCz38qKZkvuHo/Icons?node-id=55%3A2

**Comment by SiAdcock** [comment]
This looks okay to me! 👌 

I'd like to have sorted out the slightly clunky typography API before this migration, but it should be possible to do another big find and replace later.

**Pull request merged**